Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -0,0 +1,216 @@
+ports <https://docs.cilium.io/en/v1.8/install/system_requirements/#firewall-rules>`_.
+Please note that ``openshift-install`` doesn't support custom firewall
+rules, so you will need to use one of the following scripts if you are
+using AWS or GCP. Azure does not need additional configuration.
+
+.. note::
+
+   This has to be done just after ``INFO Waiting up to 40m0s for
+   bootstrapping to complete...`` appears in the logs. It is safe to apply
+   these changes once, OpenShift will not override these.
+
+.. tabs::
+
+   .. group-tab:: AWS: enable Cilium ports 
+
+      .. code:: bash
+
+         infraID="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.infraID')"
+         aws_region="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.aws.region')"
+         cluster_tag="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.aws.identifier[0] | to_entries | "Name=tag:\(.[0].key),Values=\(.[0].value)"')"
+
+         worker_sg="$(aws ec2 describe-security-groups --region "${aws_region}" --filters "${cluster_tag}" "Name=tag:Name,Values=${infraID}-worker-sg" | jq -r .SecurityGroups[0].GroupId)"
+         master_sg="$(aws ec2 describe-security-groups --region "${aws_region}" --filters "${cluster_tag}" "Name=tag:Name,Values=${infraID}-master-sg" | jq -r .SecurityGroups[0].GroupId)"
+
+         aws ec2 authorize-security-group-ingress --region "${aws_region}" \
+            --ip-permissions \
+               "IpProtocol=udp,FromPort=8472,ToPort=8472,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+               "IpProtocol=tcp,FromPort=4240,ToPort=4240,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+            --group-id "${worker_sg}"
+
+         aws ec2 authorize-security-group-ingress --region "${aws_region}" \
+            --ip-permissions \
+               "IpProtocol=udp,FromPort=8472,ToPort=8472,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+               "IpProtocol=tcp,FromPort=4240,ToPort=4240,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+            --group-id "${master_sg}"
+
+   .. group-tab:: GCP: enable Cilium ports
+
+      .. code:: bash       
+
+         infraID="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.infraID')"
+         gcp_projectID="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.gcp.projectID')"
+
+         gcloud compute firewall-rules create \
+            --project="${gcp_projectID}" \
+            --network="${infraID}-network" \
+            --allow=tcp:4240,udp:8472,icmp \
+            --source-tags="${infraID}-worker,${infraID}-master" \
+            --target-tags="${infraID}-worker,${infraID}-master" \
+              "${infraID}-cilium"
+
+Accessing the Cluster
+=====================
+
+To access the cluster you will need to use ``kubeconfig`` file from the ``${CLUSTER_NAME}/auth`` directory:
+
+.. code:: bash
+
+   export KUBECONFIG="${CLUSTER_NAME}/auth/kubeconfig"
+
+.. include:: k8s-install-connectivity-test.rst
```

Comment:
Looks like connectivity check doc won't work as is, we will need grant it some extra privileges for `hostPort` check to work.

Reply:
I have a solution, but lead me to creating `SecurityContextContstaint` for connectivity check and for Cilium itself also, which I'm keen to include, but need to validate it first.

Reply:
This has been resolved. Since Cilium itself works without SCC, I'm going to leave it out for now.